tutorials/hsdk_rt_analysis/artifact/README.md

```diff
@@ -0,0 +1,249 @@
+- `--main-experiments` or `-m`
+    - Option to run primary experiments.
+- `--overhead` or `-o`
+    - Option to run evaluation of system overheads.
+- `--scalability` or `-s`
+    - Option to run evaluation of scalability.
+
+The experiments cover a user-defined number of variations, as determined by `numvars`. To exactly match the results in the paper,
+`numvars` can be set to 0. In this case, each graph will have a number of variations equal to its node count.
+In this case, the experiments cover a total of 51 variations, each corresponding to one of 8 graphs (see figure 9 in the paper).
+The total number of iterations run over the course of the main experiments will thus be `51 * iterations`. Otherwise, 
+the total number of iterations will be `numvars * iterations`.
+
+As a result, the runtime of the experiments is almost entirely dependent on the values of these two arguments. With both `numvars` and `iterations` at 10,
+execution of the main experiments will take around 20 minutes. This scales approximately linearly, so 100 will yield an execution time of around 3 hours,
+and 1000 around 30 hours. Our experiments in the paper roughly correspond to an `iterations` value of 1000 with numvars at 0, but as few as 10 iteration
+should still yield valid results for the majority of randomly generated graph variations.
+
+Scalabilty experiments run a timing analysis of our algorithm averaging over multiple runs, so they may take around 15 minutes to complete with default settings.
+
+#### Run all experiments
+
+Before running the main or overhead experiments, it is necessary to first build the executables with the `--build` option. 
+This uses the `--iterations` argument, so the executables will need to be rebuilt if a different `--iterations` value is desired. This is necessary because Holoscan requires some application properties to be hardcoded at build time.
+
+    cd artifact-eval
+    python master.py --iterations 10 -b
+    python master.py --numvars 10 -m -o -s
+
+While running the experiments, output will periodically print to the terminal. The completion of an iteration prints the following output:
+
+    Running... iteration 1 complete
+
+Furthermore, the completion of a graph's execution will print the following output:
+
+    Graph 1 variation 0 complete
+
+The simulations done in the main and scalability experiments do not print anything to the terminal, but these only take around a minute to complete.
+
+When the experiments finish, the `data` folder will be populated with all the resulting raw data and figures, described in the Output section below.
+
+### Viewing experiment results and cleanup
+
+To transfer files from the container to the local system, copy the files to the `saved_artifacts` folder. For example:
+
+    cp data/*.pdf saved_artifacts
+
+This will make these files available even after the container is gone. When finished with the container, use `exit` to shut it down. 
+
+## Output
+
```

Comment:
Can we make it clear about what `master.py` commands to run for which results/pdfs? @pschowitz